client/src/pages/About.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import '../App.css'
+import Hero from '../components/HeroAbout';
+
+const About = () => {
+  return (
+    <div>
+      <Hero />
+      <div className="aboutcontainer">
+        <div className="row">
+          <div className="col-md-2"></div>
+          <div className="col-md-8">
+            <div className="abouttitle">
+            <h2><strong><center>It will invade your life. It will haunt your dreams. It will control you. Say YES to running.</center></strong></h2>
+            </div>
+            <p>Jog Logs was born out of necessity. While training for the 2019 New York City Marathon, Jog Logs founder
+              Aaron Phillips was looking for a training plan but was frustrated with what he found. After rebounding from an
+          injury, he had a unique goal: go from couch to marathon in 8 months. </p>
+            <p>There are plenty of one-size-fits-all
+              marathon training plans available online but they don't take into account the runner's fitness and are only
+              available as static spreadsheets. The alternative was to seek a training coach to write a custom plan but this
+          was a far more expensive option and still only yielded a static spreadsheet.</p>
+            <p>The idea for jog logs started to form over the course of a long run. It turns out, many in the running
+              community weren't happy with their options when it comes to training plans. After fielding input from the
+              running community, it became very clear that runners need dynamic training plans that are customized to their
+          unique needs and are easy to access. </p>
```

Comment:
You know 4 people worked on this, right?

Reply:
Of course!  I didn't mean to negate anyone's contribution.  I was only trying to come up with an origin story.  I've tweaked the text so that it's completely neutral.  Sorry for that.